index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
Zooming in and out seems fine to me. @Tusharmalik I do not understand the role of zooming effect in making website look vulnerable, explain a tad bit more please?